src/transaction.rs

```diff
@@ -626,6 +626,88 @@ impl From<PunishTransaction> for Transaction {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct CloseTransaction {
+    inner: Transaction,
+    input_descriptor: Descriptor<bitcoin::PublicKey>,
+    digest: SigHash,
+}
+
+impl CloseTransaction {
+    pub fn new(
+        TX_f: &FundingTransaction,
+        (amount_a, output_a): (Amount, Address),
+        (amount_b, output_b): (Amount, Address),
+    ) -> Self {
+        let output_a = TxOut {
+            value: amount_a.as_sat() - 10_000,
```

Comment:
:thinking: I believe we could do with a `5_000` satoshi fee per party here, mimicking the total fee of `10_000` for _one_ output in the `CommitTransaction`. It really doesn't matter though.

Reply:
Either case I'd prefer if there were in a `const`.

Reply:
I think fees need some more thinking. 
Are you fine with me leaving it as it is?

Reply:
Just use the global constant after you rebase.